### PR TITLE
Add Remote Control prolight

### PR DESCRIPTION
**Pull request by patatman**
Add Prolight Remote Control that comes with the GUI10 kit

### CODE REVIEW

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1222/files

File: devices/prolight.js [hunk 1]
```diff
@@ -29,4 +32,12 @@ module.exports = [
         description: 'GU10 white and colour spot',
         extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 555]}),
     },
+    {
+        zigbeeModel: ['PROLIGHT REMOTE CONTROL'],
+        model: 'REMOTE CONTROL',
```
[the remark this answers is not in the document]
Comment (patatman) [reply devices/prolight.js:37]: I had a look, but it doesn't show anything.
I had a look into the bulbs, but those showup to the coordinator as the numbers, this one just shows up as "REMOTE CONTROL"

[the remark this answers is not in the document]
Comment (patatman) [reply devices/prolight.js:37]: image

**Comment by patatman** [comment]
Thanks for the fast merge @Koenkk ! 🥳

**Pull request merged**